gen/nvml/device.go

```diff
@@ -2440,37 +2440,6 @@ func (Device Device) GetTargetFanSpeed(Fan int) (int, Return) {
 	return DeviceGetTargetFanSpeed(Device, Fan)
 }
 
-// nvml.DeviceGetPowerMode()
-func DeviceGetPowerMode(Device Device) (int, Return) {
```

Comment:
Just to clarify. These were removed.

Reply:
Yeah. I was surprised -- but I did a diff of the 11.8 and 12.0 nvml.h files and they are gone...